mcode/request-generator
# Remove unused variables and imports

## Describe your changes

> Please include a summary of the changes and the related issue/task. Please also include relevant motivation and context. List any dependencies that are required for this change, including links to other pull requests/branches in other repositories if applicable.

Removed unused code while doing [REMS-678](https://jira.mitre.org/browse/REMS-678) and looking at request-generator as a reference

## Issue ticket number and Jira link

> Please include the Jira Ticket Number and Link for this issue/task.

N/A

## Checklist before requesting a review

## Checklist for conducting a review

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you. 

# How to test manually
1. Go to settings panel
2. e.g. For TIRF, change the order-sign endpoint URL to point to rems-patient-view at the end
3. Open network tab
4. Select Jon Snow & TIRF, then click sign order
5. Notice that the rems-patient-view network request was hit and a list of cards was returned as a response
